cmd/wallet-web/package.json

```diff
@@ -44,7 +44,7 @@
     "@vue/cli-plugin-babel": "^4.1.0",
     "@vue/cli-plugin-eslint": "^5.0.0-beta.3",
     "@vue/cli-plugin-router": "^4.5.13",
-    "@vue/cli-service": "^4.1.0",
+    "@vue/cli-service": "^4.5.13",
```

Comment:
"@vue/cli-service": "^4.5.13",  I tested this i local, this change is breaking the test. Roll back the cli version and hopefully test will pass.

Reply:
This does not resolve the test issue